### PR TITLE
Update for new language.

### PR DESCRIPTION
**Pull request by squidarth**
# Summary

Update language for the new model management stuff. The main things that needed to change were model-version -> model-deployment & draft -> development.

# Testing

## Help text

```

$ poetry run truss predict --help
                                                                                                                                                                                                                                                                                          
 Usage: truss predict [OPTIONS]                                                                                                                                                                                                                                                           
                                                                                                                                                                                                                                                                                          
 Invokes the packaged model                                                                                                                                                                                                                                                               
 TARGET_DIRECTORY: A Truss directory. If none, use current directory.                                                                                                                                                                                                                     
 REQUEST: String formatted as json that represents request                                                                                                                                                                                                                                
 REQUEST_FILE: Path to json file containing the request                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                          
╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ --target-directory      TEXT  Directory of truss                                                                                                                                                                                                                                       │
│ --remote                TEXT  Name of the remote in .trussrc to push to                                                                                                                                                                                                                │
│ --data              -d  TEXT  String formatted as json that represents request                                                                                                                                                                                                         │
│ --file              -f  PATH  Path to json file containing the request                                                                                                                                                                                                                 │
│ --published                   Invoked the published model version.                                                                                                                                                                                                                     │
│ --model-version         TEXT  [DEPRECATED] Use --model-deployment instead, this will be removed in future release. ID of model deployment                                                                                                                                              │
│ --model-deployment      TEXT  ID of model deployment to invoke                                                                                                                                                                                                                         │
│ --model                 TEXT  ID of model to invoke                                                                                                                                                                                                                                    │
│ --help                        Show this message and exit.                                                                                                                                                                                                                              │
╰────────────────────────────────────────────
```
## For truss predict

With model-version

```
$ poetry run truss predict --model-version w7p6myw -d '{"a": "b", "c": true}'
? 🎮 Which remote do you want to connect to? sid-prod
[DEPRECATED] --model-version is deprecated, use --model-deployment instead.
{
  "error": "Internal Server Error"
}

```

with model-deployment

```
$ poetry run truss predict --model-deployment w7p6myw -d '{"a": "b", "c": true}'
? 🎮 Which remote do you want to connect to? sid-prod
{
  "error": "Internal Server Error"
}
```
##  For Truss push

```
 $ poetry run truss push
? 🎮 Which remote do you want to connect to? sid-prod
Compressing... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
Uploading... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
✨ Model test-201 was successfully pushed ✨

|---------------------------------------------------------------------------------------|
| Your model has been deployed as a development model. Development models allow you to  |
| iterate quickly during the deployment process.                                        |
|                                                                                       |
| When you are ready to publish your deployed model as a new version,                   |
| pass `--publish` to the `truss push` command. To monitor changes to your model and    |
| rapidly iterate, run the `truss watch` command.                                       |
|                                                                                       |
|---------------------------------------------------------------------------------------|

🪵  View logs for your deployment at https://app.baseten.co/models/7wl1n91q/versions/w7p6myw/logs
```

### CODE REVIEW

File: truss/cli/cli.py [hunk 1]
```diff
@@ -304,7 +306,13 @@ def _extract_request_data(data: Optional[str], file: Optional[Path]):
...
     type=str,
     required=False,
-    help="ID of model version to invoke",
+    help="[DEPRECATED] Use --model-deployment instead, this will be removed in future release. ID of model version",
+)
+@click.option(
+    "--model-deployment",
+    type=str,
+    required=False,
+    help="ID of model deployment to invoke",
```
Comment (philipkiely-baseten) [inline truss/cli/cli.py:315]: We standardized on "call" instead of "invoke" in the product, would be good to match here.

File: truss/cli/cli.py [hunk 2]
```diff
@@ -419,7 +435,7 @@ def push(
     if service.is_draft:
         draft_model_text = """
 |---------------------------------------------------------------------------------------|
-| Your model has been deployed as a draft. Draft models allow you to                    |
+| Your model has been deployed as a development model. Development models allow you to  |
 | iterate quickly during the deployment process.                                        |
 |                                                                                       |
 | When you are ready to publish your deployed model as a new version,                   |
```
Comment (philipkiely-baseten) [inline truss/cli/cli.py:441]: This still says "version", I suggest something like `When you are ready to publish a new deployment of your model, pass`...

Comment (squidarth) [reply truss/cli/cli.py:441]: good catch, thanks!

File: truss/cli/cli.py (hunk 1 above)
Comment (squidarth) [reply truss/cli/cli.py:315]: cool will change

**Review by moward — APPROVED** [review]

**Review by bolasim — APPROVED** [review]

**Pull request merged**